sdks/wasm/DebuggerTestSuite/BreakpointTest.cs

```diff
@@ -0,0 +1,89 @@
+using System;
```

Comment:
We should also add tests to specifically test the `column`, and `end` being optional.

Reply:
I'm calling `Debugger.getPossibleBreakpoints` without any of these in here.  I could add some tests with `column` or `end`, though.